src/plugins/kibana/public/settings/sections/about/index.html

```diff
@@ -2,7 +2,7 @@
   <div class="kbn-settings-about container" ng-controller="settingsAbout">
     <div class="col-md-4 col-md-offset-4 jumbotron">
       <center>
-        <img src='/plugins/kibana/settings/sections/about/barcode.svg' alt="Kibana Barcode Logo" width="128" height="128"><br>
+        <img src='../plugins/kibana/settings/sections/about/barcode.svg' alt="Kibana Barcode Logo" width="128" height="128"><br>
```

Comment:
What do you think about using a base url html attribute in kibana rather than trying to do path traversal on assets?